src/ios-deploy/ios-deploy.m

```diff
@@ -3113,6 +3113,28 @@ void kill_app(AMDeviceRef device) {
     [ns_pid release];
 }
 
+void launch_app(AMDeviceRef device) {
+    if (bundle_id == NULL ) {
+        on_error(@"Error: must specify --bundle_id");
```

Comment:
Nitpicking, but it could be good to be consistent with the similar `bundle_id` missing error message [above](https://github.com/ios-control/ios-deploy/pull/587/files#diff-dfd715eb4a0f5d4f337914719ad64dc88bd3f6e6c8aff5c03f8c38b524bdb31fR3076). Though I see another message follows this format, so feel free to ignore. 

```suggestion
        on_error(@"Error: bundle id required, please specify with --bundle_id.");
```